static/css/tools/choropleth.scss

```diff
@@ -50,22 +50,24 @@
 #breadcrumbs {
   float: left;
   height: 30px;
-  width: 40%;
+  width: 100%;
   margin-left: 15px;
 }
 
 #hover-text-display {
   float: right;
   width: 40%;
+  height: 0;
 }
 
 #heading {
   font-size: 32px;
   text-align: center;
+  width: 100%;
```

Comment:
div's are usually 100% width by default, so perhaps it's the column class that's doing something here.

Reply:
Oh you're right, this was redundant. Removed.